src/view/details.js

```diff
@@ -0,0 +1,131 @@
+define(["text!../template/details.html"], function (html) {
+
+    var body = document.body;
+
+    // Load the HTML
+    var template = document.createElement("body");
+    template.innerHTML = html;
+
+    // Itinerary
+    var title = template.querySelector("h2");
```

Comment:
Good! When I started to do it I thought in avoid to use jQuery, but I tested the request with jQuery and I forget to remove it after, converting to XMLHttpRequest. Was lack of attention, my fault again!